internal/cmd/tui.go

```diff
@@ -44,7 +44,13 @@ func tuiCmd() *cobra.Command {
 			blocks = sortBlocks(blocks)
 
 			if len(blocks) == 0 {
-				return errors.Errorf("no code blocks in %s", fFileName)
+				if fFileMode {
+					return errors.Errorf("no code blocks in %s", fFileName)
+				}
+				if !fAllowUnnamed {
+					return errors.Errorf("no named code blocks, consider adding flag --allow-unnamed")
+				}
+				return errors.Errorf("no named code blocks")
```

Comment:
```suggestion
				return errors.Errorf("no code blocks")
```

At this point, `fAllowUnnamed` is true. So there are no named or unnamed code blocks.

Reply:
Am I crazy? I just tested this. Let me look again.